msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -151,21 +151,20 @@
     <None Update="Platforms\net45\SilentWindowsFormsAuthenticationDialog.cs" SubType="Form" />
   </ItemGroup>
 
-  <!-- Copy all artifacts needed to generate the nuget package. This should be replaced with msbuild /t:pack, however
+  <!-- Copies all artifacts needed to generate the nuget package. This should be replaced with msbuild /t:pack, however
   bug https://github.com/NuGet/Home/issues/3891 blocks this, espcially that packaging rebuilds the Core assemblies, so we cannot sign it.-->
   <Target Name="LayoutNuGet">
     <PropertyGroup>
-      <NugetPackageRootFolder Condition="'$(NugetPackageRootFolder)' == ''">$([System.IO.Path]::Combine($(MSBuildThisFileDirectory), "..", "..", "layout"))</NugetPackageRootFolder>
-      <PackagePdb Condition="'$(PackagePdb)'==''">false</PackagePdb>
+      <NugetPackageRootFolder Condition="'$(NugetPackageRootFolder)' == ''">$([System.IO.Path]::Combine($(MSBuildThisFileDirectory), "..",".nugetlayout"))</NugetPackageRootFolder>
       <NuspecFilePath>$([System.IO.Path]::Combine($(MSBuildThisFileDirectory), "..", "..", "build", "Microsoft.Identity.Client.nuspec"))</NuspecFilePath>
     </PropertyGroup>
     <ItemGroup>
       <FilesToPack Include="bin\$(Configuration)\**\Microsoft.Identity.Client.dll" />
       <FilesToPack Include="bin\$(Configuration)\**\Microsoft.Identity.Client.xml" />
-      <FilesToPack Include="bin\$(Configuration)\**\Microsoft.Identity.Client.pdb" Condition="'$(PackagePdb)' == 'true' " />
+      <FilesToPack Include="bin\$(Configuration)\**\Microsoft.Identity.Client.pdb" />
```

Comment:
Did we decide to systematically package the PDBs? (I'm all for it, BTW)

Reply:
The packaging task are creating 2 packages - one with DLL and XML, and one with DLL, PDB and XML. I figured this behaviour is what we always want and there's no need to control copying the pdbs.